.gitignore

```
@@ -9,3 +9,7 @@
 
 # misc
 .DS_Store
+cypress.env.json
+cypress/downloads/
+cypress/screenshots/
+cypress/videos/
```

É uma boa prática deixar uma linha em branco ao final de cada arquivo, para que o código seja melhor renderizado em serviços como o GitHub, onde quando não há tal linha em branco, um ícone é exibido ao final de cada arquivo, informando No newline at end of file.

```suggestion
cypress/videos/

```